mlir/lib/Driver/CompilerDriver.cpp

```diff
@@ -861,26 +865,30 @@ int QuantumDriverMainFromCL(int argc, char **argv)
     // ---------
     // Any modifications made to the command-line interface should be documented in
     // doc/catalyst-cli/catalyst-cli.rst
-    cl::opt<std::string> WorkspaceDir("workspace", cl::desc("Workspace directory"), cl::init("."));
+    cl::OptionCategory CatalystCat("Catalyst-cli Options", "Frist print catalyst-cli options");
```

Comment:
Don't think we need this string after looking at the output
```suggestion
    cl::OptionCategory CatalystCat("Catalyst-cli Options", "");
```